app/controllers/admin/base_admin_controller.rb

```diff
@@ -1,11 +1,25 @@
 module Admin
   class BaseAdminController < ApplicationController
-    before_action :ensure_authenticated_user
+    TIMEOUT_LENGTH_IN_MINUTES = 30
+
+    before_action :ensure_authenticated_user, :end_expired_sessions, :update_last_seen_at
```

Comment:
Is `update_last_seen_at` in the `ApplicationController` now?

Reply:
Yeah, it was moved there as part of the Verify work